app/redux/reducers/SetsReducer.js

```diff
@@ -93,6 +97,27 @@ const createDefaultState = () => {
     };
 };
 
+// Set default metric
+
+const setDefaultMetric = (state, action) => {
```

Comment:
Rename the function and the comment to saveDefaultMetric

Reply:
Done